buzzmobile/drive/horn/src/horn_node.cpp

```diff
@@ -0,0 +1,42 @@
+#include <ros/ros.h>
+#include <std_msgs/Bool.h>
+#include <buzzmobile/CarPose.h>
```

Comment:
This doesn't seem to compile:
https://travis-ci.org/gtagency/buzzmobile/builds/168462865

Maybe we need to `<depend>` on the manifest?